tsconfig.json

```diff
@@ -30,5 +26,7 @@
     ],
     "exclude": [
         "node_modules"
-    ]
+    ],
+    
+    "extends": "./.nuxt/tsconfig.json"
```

Comment:
let's move this to the top of the file. It makes it easier to the next reader to know we're building on rules from a base config.